# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/antitracking/bouncetrackingprotection/moz.build
# Commit: 45ea26477050
# Full Hash: 45ea26477050c8d3e4c6565ad182892159323e1c
# Author: Paul Zuehlcke <pbz@mozilla.com>
# Date: 2023-09-19 03:53:26
# Regressor Bug: 1839918
# File Overlap Count: 1
# Description:
#   Bug 1839918 - Add BounceTrackingStorageObserver to observe cookie changes. r=bvandersloot,anti-tracking-reviewers
#   
#   Depends on D186340
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D185304
# ==============================================================================

diff -r c7862a00e6e7 -r 45ea26477050 toolkit/components/antitracking/bouncetrackingprotection/moz.build
--- a/toolkit/components/antitracking/bouncetrackingprotection/moz.build	Mon Sep 18 11:55:10 2023 +0000
+++ b/toolkit/components/antitracking/bouncetrackingprotection/moz.build	Mon Sep 18 11:55:10 2023 +0000
@@ -22,12 +22,14 @@
     "BounceTrackingProtection.h",
     "BounceTrackingRecord.h",
     "BounceTrackingState.h",
+    "BounceTrackingStorageObserver.h",
 ]
 
 UNIFIED_SOURCES += [
     "BounceTrackingProtection.cpp",
     "BounceTrackingRecord.cpp",
     "BounceTrackingState.cpp",
+    "BounceTrackingStorageObserver.cpp",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")
